roles/common/tasks/banner.yaml

```
@@ -0,0 +1,13 @@
+---
+
```

See here https://github.com/Kv-126-DevOps/ansible/blob/main/roles/common/molecule/default/converge.yml#L4
So, need to add
```suggestion
- name: Gather metadata facts
  setup:
    gather_subset:
      - '!all'
      - hardware

```